src/library/Book.java

```diff
@@ -0,0 +1,68 @@
+package library;
+
+
+import java.util.Objects;
+
+public class Book implements Comparable<Book> {
+
+    private int id;
+    private String title;
+    private String genre;
+
+    public Book(int id, String title, String genre) {
+        this.id = id;
+        this.title = title;
+        this.genre = genre;
+    }
+
+    public int getId() {
+        return id;
+    }
+
+    public void setId(int id) {
```

Comment:
we should avoid possibility to update IDs

Reply:
Done